tests/test_visitors/test_ast/test_complexity/test_function/test_elifs.py

```diff
@@ -8,7 +8,7 @@
 )
 
 function_with_elifs = """
-def test_module():
+{0} def test_module():
```

Comment:
It should be `{0}def`. Without an extra space.
The same applies to all other fixtures.